R/bootstrap.R

```diff
@@ -16,11 +30,19 @@ as_strata <- function(...){
 }
 
 
-#' Title
+#' Sample Patient Ids
+#'
+#' Performs stratified random sampling with replacement of patient IDs
+#' ensuring the return vector is the same length as the input vector
```

Comment:
I would maybe add "That is, performs a stratified bootstrap sample of ids"?

Reply:
Done